source/recipes/product-vulnerability-incident.rst

```diff
@@ -0,0 +1,112 @@
+Mattermost Recipe: How to use Mattermost for Product Vulnerability Incidents
+=====================
+
+Problem
+---------------
+
+After getting access to your Mattermost instance, you’d like to set up a workflow for security incident handling using
+Mattermost. Click `here<https://mattermost.com/deploy/>`__ if you want to learn more about how to deploy your own Mattermost instance.
+
+Solution
+---------------------
+
+The following guide will walk through the set up of a product security incident room using the Mattermost `Playbooks<https://docs.mattermost.com/guides/playbooks.html>`__ and
+`Channels<https://docs.mattermost.com/guides/channels.html>`__ with the voice calling and screen sharing functionalities.
+
+1. Workspace Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For Cloud customers, all the functionalities will work out-of-the-box with no technical setup. Simply invite your team
+members to your workspace and move onto the next section For self-hosted installations, refer to `this admin guide<https://docs.mattermost.com/configure/configuration-settings.html#calls-beta>`__ to
+configure voice calling and screen sharing functionalities.
+
+2. Playbooks Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now that your workspace and teammates are ready, the next step is to create a playbook for incident handling. For this
+recipe, we will be using an adapted playbooks template used by the Mattermost Product Security Team that is available for
+`download here<https://github.com/mattermost/mattermost-security/blob/master/product_security/playbooks/security_vulnerability_playbook.json>`__.
+
+After downloading the template above, open your Mattermost instance and click in the top-left menu on “Playbooks”:
+
+.. image:: ../images/recipe/prod-vuln-incident1.png
+   :alt: ..
+
+Afterwards, open the “Playbooks” menu and click the “Import” button, where you can then navigate to the file
+downloaded above:
+
+.. image:: ../images/recipe/prod-vuln-incident2.png
+   :alt: ..
+
+After importing the file, you should have a new playbook called “Security Vulnerability Playbook”. Click on the three
+dots and hit “Edit”:
+
+.. image:: ../images/recipe/prod-vuln-incident3.png
+   :alt: ..
+
+Modify any tasks or actions as you see fit. In our scenario, we’ve assigned the playbook owner role by default to
+“alice” and added 3 other team members to be automatically added in the actions tab. We recommend changing these to the
+members of your security team:
+
+.. image:: ../images/recipe/prod-vuln-incident4.png
+   :alt: ..
+
+Afterwards, click Save and you are good to go. Hit the “Run” button on the Playbook:
+
+.. image:: ../images/recipe/prod-vuln-incident5.png
+   :alt: ..
+
```

Comment:
Optional suggestion (just because I missed it, I am assuming others might too 😃 ): 

Can we move the point `Hit the “Run” button on the Playbook:` as a separate next step. The step `Afterwards, click Save and you are good to go.` is too intriguing by itself to indicate that that step is complete.

Reply:
I think thanks to @justinegeffen review we got this point covered, agreed? :-)